pkg/products/fuse/reconciler.go

```diff
@@ -3,6 +3,9 @@ package fuse
 import (
 	"context"
 	"fmt"
+	"github.com/integr8ly/cloud-resource-operator/pkg/apis/integreatly/v1alpha1/types"
```

Comment:
Should we stick to calling this `croTypes` as you do in the test https://github.com/integr8ly/integreatly-operator/pull/597/files#diff-3f2c31aa1bb5bff6fd6d99ecad290b71R13

Reply:
Sure

Reply:
:+1: